packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { withPrefix } from "gatsby"
+import * as React from "react"
+
+import { cx } from "../../lib/cx"
+import { SearchHit } from "./types"
+
+import "./ResultRow.scss"
+
+export type ResultRowprops = {
+  exactMatch?: boolean
+  installer: [string, string]
+  hit: SearchHit
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
+  exactMatch,
+  installer,
+  hit: {
+    description,
+    downloadsLast30Days,
+    humanDownloadsLast30Days,
+    modified,
+    name,
+    types,
+  },
+}) => {
+  const [icon, label] =
+    types.ts === "included"
+      ? ["in", "included"]
+      : ["dt", "from DefinitelyTyped"]
+
+  return (
+    <tr className={cx("resultRow", exactMatch && "resultRowExactMatch")}>
+      <td
+        className={cx(
+          "downloads",
+          downloadsLast30Days > 1_000_000
+            ? "downloadsMillion"
+            : downloadsLast30Days > 100_000 && "downloadsHundredThousand"
+        )}
+      >
+        {humanDownloadsLast30Days}
+      </td>
+      <td className="via">
+        <div
+          aria-label={`Types ${label}`}
+          className={cx("typeImage", icon)}
+          role="img"
+        />
+      </td>
+      <td className="name">
+        <strong>{name}</strong>
+        {description}
+      </td>
+      <td className="updated">{modified}</td>
```

Comment:
Question: do you have a preference for how to convert the updated timestamp to human text? My default would be `moment` from past usage, but it is a little large...

Reply:
Aye, use the platform, I use `Intl` in a few places now:

```ts
  const d = new Date(props.lastEdited);
  const dtf = new Intl.DateTimeFormat(
    props.lang,
    { year: "numeric", month: "short", day: "2-digit" },
  );
  const lastEdited = dtf.format(d);
```

```ts
  const releaseParts = intl.formatDateToParts(startDate, { month: "short", day: "numeric" })
```

Reply:
Love it 😍